test_data/of15/group_add.data

```diff
@@ -0,0 +1,40 @@
+-- binary
+06 0f # version, type
+00 38 # length
+12 34 56 78 # xid
+00 00 # command (OFPGC_ADD)
+01 # type (OFPGT_SELECT)
+00 # pad
+00 00 00 01 # group_id
+00 01 # bucket_array_len
+00 00 # pad
+ff ff ff 00 # command_bucket_id (BUCKET_MAX)
+
+
+
+00 20 # len
+00 0a # action_array_len
+ff ff ff fd # bucket_id
+
+
+00 00 # ofp_action[0].type (OFPAT_OUTPUT)
+00 10 # ofp_action[0].len
+00 00 00 01 # port
+00 03 # max_len
+00 00 00 00 00 00 # pad
+
+00 16 # of_action[0].type (OFPAT_GROUP)
+00 08 # of_action[0].length
+00 00 00 01 # group_id
```

Comment:
Need to also add in ofp_group_bucket_prop_weight for SELECT group type. Doing so (and correcting lengths now results in:
testRead(org.projectfloodlight.openflow.protocol.ver15.OFGroupAddVer15Test): Unknown value for discriminator type of class OFGroupPropVer15: 0

however OF1.5 spec defines OFPGBPT_WEIGHT as 0. Need to look into this next